modules/firelens_log_config/main.tf

```diff
@@ -0,0 +1,136 @@
+# This is just a module to provide a template for creating a log configuration for an ECS task definition.
+
+locals {
+  tld = var.region == "eu-central-1" ? "eu" : "com"
+
+  ecs_shared_env_vars = <<EOF
```

Comment:
How about these ones? I think we'd need these in each module but not here.

Reply:
Yeah we can remove these I think